packages/framework/presence/src/latestMapValueManager.ts

```diff
@@ -489,6 +489,24 @@ class LatestMapValueManagerImpl<
 	}
 }
 
+/**
+ * Arguments that are passed to the {@link StateFactory.latestMap} function.
+ *
+ * @alpha
+ */
+export interface LatestMapArguments<
+	T extends object,
+	Keys extends string | number = string | number,
+> {
+	/**
+	 * The initial value of the local state.
+	 */
+	local?: {
+		[K in Keys]: JsonSerializable<T> & JsonDeserialized<T>;
+	};
+	settings?: BroadcastControlSettings | undefined;
```

Comment:
Nit (same for `LatestArguments`), so it doesn't go undocumented. The docs on the type feel like the thing one would want to see, and copying them in each param doesn't make sense IMO.
```suggestion
	/**
	 * See {@link BroadcastControlSettings}.
	 */
	settings?: BroadcastControlSettings | undefined;
```